compiler-rt/lib/radsan/radsan_user_interface.cpp

```diff
@@ -32,23 +32,21 @@ std::function<OnErrorAction()> createErrorActionGetter() {
       return OnErrorAction::Continue;
   };
 
-  auto user_mode = __sanitizer::GetEnv("RADSAN_ERROR_MODE");
-  if (user_mode == nullptr) {
-    return exit_getter;
-  }
+  CHECK(radsan::IsInitialized());
```

Comment:
What happens if it's not initialised here?

Reply:
Process aborts, as in a failed assertion.

However, if we remove this `CHECK` the line on 36 aborts because `radsan::flags()` is nullptr. This just gives a better name to the problem, that we are not initialized so we should be dying here.